### PR TITLE
add openssl vuln policy

### PR DESCRIPTION
**Pull request by atomic111**
Signed-off-by: Patrick Münch <patrick.muench1111@gmail.com>

### CODE REVIEW

File: core/mondoo-unix-openssl-vulnerability.mql.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Read more about the policy structure at https://mondoo.com/docs/platform/policies/overview
+policies:
```
Comment (tas50) [inline core/mondoo-unix-openssl-vulnerability.mql.yaml:2]: We should include a general policy description since that shows up in the Policy Hub.

**Review by chris-rock — APPROVED** [review]
Thank you @atomic111

**Review by tas50 — APPROVED** [review]

File: core/mondoo-unix-openssl-vulnerability.mql.yaml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    desc: |
+      ## Overview
+
+      OpenSSL Vulnerability Policy by Mondoo checks for vulnerable OpenSSL installation on Unix/ Linux system.
+
+      ## Remote scan
+
+      Remote scans use native transports in `cnspec` to provide on demand scan results without the need to install any agents, or integration.
+
+      For a complete list of native transports run:
```
Comment (chris-rock) [inline core/mondoo-unix-openssl-vulnerability.mql.yaml:23]:
```suggestion
      For a complete list of native providers run:
```

File: core/mondoo-unix-openssl-vulnerability.mql.yaml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+          mondoo-openssl-vulnerability:
+    docs:
+    desc: |
+      ## Overview
+
+      OpenSSL Vulnerability Policy by Mondoo checks for vulnerable OpenSSL installation on Unix/ Linux system.
+
+      ## Remote scan
+
+      Remote scans use native transports in `cnspec` to provide on demand scan results without the need to install any agents, or integration.
```
Comment (chris-rock) [inline core/mondoo-unix-openssl-vulnerability.mql.yaml:21]:
```suggestion
      Remote scans use native providers in `cnspec` to provide on demand scan results without the need to install any agents, or integration.
```

**Pull request merged**